app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/SettingsActivity.java

```diff
@@ -283,9 +301,32 @@ void invalidateSettings() {
                 });
             }
 
-            final ATEColorPreference primaryColorPref = findPreference("primary_color");
+            final TwoStatePreference rememberLastTab = findPreference(PreferenceUtil.REMEMBER_LAST_TAB);
+            if (rememberLastTab != null) {
+                rememberLastTab.setChecked(PreferenceUtil.getInstance().rememberLastTab());
+                rememberLastTab.setOnPreferenceChangeListener((preference, newValue) -> {
+                    // Save preference
+                    PreferenceUtil.getInstance().setRememberLastTab((Boolean) newValue);
+
+                    return true;
+                });
+            }
+
+            final TwoStatePreference whitelistEnabled = findPreference(PreferenceUtil.WHITELIST_ENABLED);
+            if (whitelistEnabled != null) {
+                whitelistEnabled.setChecked(PreferenceUtil.getInstance().getWhitelistEnabled());
+                whitelistEnabled.setOnPreferenceChangeListener((preference, newValue) -> {
+                    // Save preference
+                    //PreferenceUtil.getInstance().setRememberShuffle((Boolean) newValue);
+
+                    return true;
+                });
+            }
+
+            final ATEColorPreference primaryColorPref = findPreference(ThemeStore.KEY_PRIMARY_COLOR);
```

Comment:
As I mentioned in the description of the pr I have some major issues with letting the app know, that the primary, accent color, or general theming has changed after importing.